src/DynamoCore/Migration/Migration.cs

```diff
@@ -1360,6 +1354,7 @@ public class NodeMigrationAttribute : Attribute
         /// <summary>
         /// Version this migrates to.
         /// </summary>
+        [Obsolete("All migrations are now processed, beginning with the first migration whose version is >= the workspace version. The To property will be ignored.", false)]
```

Comment:
There appears to be only one `NodeMigration` applied in the Revit repos: https://github.com/DynamoDS/DynamoRevit/search?utf8=✓&q=NodeMigration